be/src/storage/tablet_updates.cpp

```diff
@@ -1959,9 +1975,11 @@ Status TabletUpdates::_commit_compaction(std::unique_ptr<CompactionInfo>* pinfo,
     std::unique_ptr<EditVersionInfo> edit_version_info = std::make_unique<EditVersionInfo>();
     edit_version_info->version = EditVersion(edit_version_pb->major_number(), edit_version_pb->minor_number());
     edit_version_info->creation_time = creation_time;
+    edit_version_info->gtid = rowset->rowset_meta()->gtid();
```

Comment:
the input rowsets in primary table compaction may be discontinuous, so the output rowset `gtid` may be inaccurate.
the `gtid` in edit_version_info should be unchanged after compaction.